src/convertToObject.js

```diff
@@ -6,7 +6,28 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceObject = {};
+
+  sourceString.split(';').forEach((line) => {
+    const trimmedLine = line.trim();
+
+    if (trimmedLine === '') {
+      return;
+    }
+
+    const colonIndex = trimmedLine.indexOf(':'); // Corrected this line
+
+    if (colonIndex === -1) {
+      return;
+    }
+
+    const property = trimmedLine.slice(0, colonIndex).trim();
+    const value = trimmedLine.slice(colonIndex + 1).trim();
+
+    sourceObject[property] = value;
+  });
+
+  return sourceObject;
```

Comment:
The function 'convertToObject' correctly splits the string by semicolons and processes each line to create a CSS properties object. However, it could be improved by using more functional programming practices such as 'reduce'.